www/_posts/2024-05-23-cordova-android-13.0.0.md

```diff
@@ -0,0 +1,158 @@
+---
+layout: post
+author:
+    name: Bryan Ellis
+title:  "Cordova Android 13.0.0 Released!"
+categories: announcements
+tags: news releases
+---
+
+We are happy to announce that we have just released `Cordova Android 13.0.0`! This is one of Cordova's supported platforms for building Android applications.
+
+* [cordova-android@13.0.0](https://www.npmjs.com/package/cordova-android)
+
+**To upgrade:**
+
+```bash
+cordova platform remove android
+cordova platform add android@13.0.0
+```
+
+**To install:**
+
+```bash
+cordova platform add android@13.0.0
+```
+
+## Release Highlights
+
+### BREAKING CHANGES
+
+* **Increased Target SDK**
+
+    This release has increased the target SDK to 34 (Android 14).
+
+* **Increased Android Studio Requirement**
+
+    Android Studio Jellyfish is required if building and running from Android Studio.
+
+* **Required Build Tools**
+
+    To use `cordova-android@13`, SDK Platform `34` and SDK Build Tools `34.0.0` must be installed. Older build tools version can be uninstalled if older versions of cordova-android is no longer used in your projects.
+
+    To install SDK Platform 34:
+
+    1. Open Android Studio's **SDK Manager**:
+    2. Click on `SDK Platforms` tab
+    3. Check `Android 14.0 ("UpsideDownCake")` which has the `API Level` of `34`
+    4. Click `Apply`
+
+    <img src="{{ site.baseurl }}/static/img/blog/2024/cordova-android-13-platform-sdk-34.png" style="width: 100%;" alt="Android SDK Platform" />
+
+    To install SDK Build Tools 34.0.0:
+
+    5. Open Android Studio's **SDK Manager**:
```

Comment:
Feels a bit weird to me that the numbering keeps going from the previous list. Maybe this should start at 1 again?